### DISCUSSION TITLE
Smooth Countdown Transition During Map Breaks

### DISCUSSION THREAD

**Discussion by ajsem1337** [ppy/osu#31019] [Ideas] [answered]
The current countdown during map breaks in osu! uses a "stepped" transition between numbers (3, 2, 1), which feels abrupt and jarring. It would be great to implement a smooth transition for the countdown bar to make it more visually consistent and natural during gameplay.

https://github.com/user-attachments/assets/c6138c27-f7dd-4e14-9a6d-ba65e311e209

**Comment by N0Y0U2114** [comment, marked as answer]
It actually used to be smooth, it got changed to this. I would think it should be a skinnable behaviour in the future.

it got added here https://github.com/ppy/osu/pull/29616

**Reply by ajsem1337** [reply]
Adding an option to change this behavior – either as a skinnable element or a setting – would be great
